### DISCUSSION TITLE
No endpoint when starting grpc client

### DISCUSSION THREAD

**Discussion by nzjcnjzx** [wechaty/wechaty#2080] [Q&A] [answered]
🚨 The issue tracker is not for questions 🚨

If you have a question, please ask it on https://stackoverflow.com/questions/tagged/wechaty

[question]
![image](https://user-images.githubusercontent.com/31735124/98753799-28eaab80-2400-11eb-812d-ce9d889c814e.png)
一直出现连接超时的问题，无法进入登录？

**Comment by huan** [comment, marked as answer]
The above message shows that the token from your [Wechaty Puppet Service](https://wechaty.js.org/docs/puppet-services/) is out of service.

To check your puppet service status by your token, you can visit <https://api.chatie.io/v0/hosties/__YOUR_TOKEN__> to check if it's available:

1. If you can get a valid ip/port then it's on service
1. If you get both ip/port with value 0 then it's out of service. Please contact your puppet service provider for support.
